tool/csv_writer_gobuster.go

```diff
@@ -0,0 +1,67 @@
+package tool
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func remove(slice []string, s int) []string {
+	return append(slice[:s], slice[s+1:]...)
+}
+
+func changePrefix(line string) string {
+	rUp, _ := regexp.Compile(`+\[[0-9]+m([A-Z]+)++\[0m`)
+	rLow, _ := regexp.Compile(`+\[[0-9]+m([a-z]+)++\[0m`)
+	word := strings.Fields(line)
+	for index := range word {
+		if index >= len(word) {
+			break
+		}
+		if rUp.MatchString(word[index]) {
+			tmp := rUp.FindStringSubmatch(word[index])
+			word = remove(word, index)
+			word = append(word[:index+1], word[index:]...)
+			word[index] = tmp[1]
+		}
+		if rLow.MatchString(word[index]) {
+			tmp := rLow.FindStringSubmatch(word[index])
+			word = remove(word, index)
+			word = append(word[:index+1], word[index:]...)
+			word[index] = tmp[1]
+		}
+	}
+	return strings.Join(word, " ")
+}
+
+func CsvWriterGobuster(record []string) {
+	data, err := os.Create("scan_data.csv")
+	if err != nil {
+		fmt.Print(err)
+	}
+	defer data.Close()
+	gobusterOut := strings.Split(record[2], "\n")
+	line := 4
```

Comment:
Make it as a constant.

Reply:
Line cannot be constant because it is increment in the for loop

Reply:
The value can be.